tests/contrib/pyodbc/test_pyodbc.py

```diff
@@ -282,6 +282,8 @@ def test_patch_unpatch(self):
             assert span.service == "pyodbc"
             assert span.name == "pyodbc.query"
             assert span.span_type == "sql"
+            assert span.get_tag("db.system") == "SQLite"
+            assert span.get_tag("db.user") == ""
```

Comment:
nit: can we update this test so that it sets a user name

Reply:
Doing so requires changing the test harness to use a database other than sqlite. Do you think that's worth the effort?